deploy/helm/grimmory/templates/pvc.yaml

```
@@ -1,11 +1,12 @@
 {{- if (and .Values.persistence.dataVolume.enabled (not .Values.persistence.dataVolume.existingClaim)) }}
+---
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   annotations:
     helm.sh/resource-policy: keep
   labels:
-    app.kubernetes.io/name: {{ include "grimmory.fullname" . }}
+    {{- include "grimmory.labels" . | nindent 4 }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)